### PR TITLE
Add the Intro section

### LINKED ISSUE #8: Add the Intro section
## GOAL

- Create a section to introduce me with a brief description

## DEADLINE / TASK SIZE

1 day

## STEPS

- Create a section with a heading of my name, my profile image, and a brief description of myself

## Design

- Mobile (~ 750px)
Image

- Tablet / Laptop (751px ~)
Image

### PR DESCRIPTION
**Pull request by HidemichiShimura**
#### Code changes
- Installed styled-components
- Added the Main component
    - `src/components/main/Main.jsx`
- Added the Intro component
    - `src/components/main/intro/Intro.jsx`

#### Issues affected

- Fixes #8 

## Screenshots and/or video

- Mobile (~ 750px)
Image

- Tablet / Laptop (751px ~)
Image

## Testing Plan

- Check the design in a web browser in the pixels below
    - Mobile View (~ 750px)
    - Tablet and Laptop View (751px ~)

#### Basics

- [x] Unnecessary commented-out code removed
- [x] Unused code removed (imports, functions, styles, etc - if it's not used anywhere, it's not in this PR)

### CODE REVIEW

File: src/components/main/intro/Intro.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    height: 150px;
+    border-radius: 50%;
+    box-shadow: 0px 0px 5px 1px var(--accent-color);
+
+    @media only screen and (min-width: 751px) {
+        width: 250px;
+        height: 250px;
+        border-radius: 50%;
+    }
+`;
```
Comment (yuya-minamide) [inline src/components/main/intro/Intro.jsx]: I have found the image collapsed.
You might try using background-image.

File: src/components/main/Main.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import styled from "styled-components";
+
+const MainSection = styled.main`
+    width: 100%;
+
+    @media only screen and (min-width: 751px) {
+        padding: 0 10%;
+    }
+`;
```
Comment (yuya-minamide) [inline src/components/main/Main.jsx]: I think the design would look better if you added margin on the left and right

File: src/components/main/intro/Intro.jsx (hunk 1 above)
Comment (HidemichiShimura) [reply src/components/main/intro/Intro.jsx]: Thank you for noticing the collapse and sharing such a good tip!
I fixed it by using the div tag and the background-image property.

File: src/components/main/Main.jsx (hunk 2 above)
Comment (HidemichiShimura) [reply src/components/main/Main.jsx]: I added horizontal padding to the Main component.
Now the mobile view should look better.

**Review by yuya-minamide — APPROVED** [review]

**Pull request merged**